starport/pkg/fswatcher/fswatcher.go

```diff
@@ -110,12 +116,18 @@ func (w *watcher) listen() {
 	}
 }
 
-func (w *watcher) addPaths(paths ...string) {
+func (w *watcher) addPath(paths ...string) {
 	for _, path := range paths {
 		w.wt.AddRecursive(filepath.Join(w.workdir, path))
```

Comment:
I think we don't need this much change in the `fswatcher`. Folowing should work already.

```go
func (w *watcher) addPaths(paths ...string) {
  for _, path := range paths {
    if !filepath.IsAbs(path) {
      path = filepath.Join(w.workdir, path)
    }
    w.wt.AddRecursive(path)
  }
}
```